code-challenges.test.js

```diff
@@ -2,58 +2,232 @@
 
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-// --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multiplied by 3.
+// =============================================================================
+// INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and
+// returns an array with all the numbers multiplied by 3.
+// =============================================================================
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
+const numbersArray1 = [6, 7, 8, 9, 10];
 // Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
+const numbersArray2 = [24, 27, 30, 33, 36];
 // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
 
-// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
-
+// =============================================================================
+// --------------------1) Create a function that takes a object as an argument
+// and decides if the number inside it is evenly divisible by three or not.
+// =============================================================================
 // a) Create a test with expect statements for each of the variables provided.
-
-const object1 = { number: 15 }
+// =============================================================================
+
+//-------------------------== Test Explanation ==-------------------------------
+// I decided to be more descriptive with this test by using multiple
+// it-statments to show different edge case tests in the terminal when Jest
+// performs tests.
+//
+// ------ Initial Test: ReferenceError: divByThree is not defined --------------
+//------------------------------------------------------------------------------
+
+describe("divByThree", () => {
+  it("returns positive object number is divisable by three..", () => {
+    expect(divByThree(object1)).toEqual("15 is divisible by three");
+  });
+  it("returns that the object number 0 is divisable by three..", () => {
+    expect(divByThree(object2)).toEqual("0 is divisible by three");
+  });
+  it("returns the negative object number is not divisable by three..", () => {
+    expect(divByThree(object3)).toEqual("-7 is not divisible by three");
+  });
+  it("returns the negative object number is divisable by three..", () => {
+    expect(divByThree(object4)).toEqual("-9 is divisible by three");
+  });
+  it("returns positive object number is not divisable by three..", () => {
+    expect(divByThree(object5)).toEqual("10 is not divisible by three");
+  });
+});
+
+// -----------------=== Provided Test Cases ===---------------------------------
+const object1 = { number: 15 };
```

Comment:
I like that you are adding edge cases and tests to account for them!  Since these test variables are only being utilized within your test, it is best practice to place the variables in your test so they are only accessible within the scope of the test.